### DISCUSSION TITLE
Set up Github Sponsorships

### DISCUSSION THREAD

**Discussion by onedr0p** [cross-seed/cross-seed#468] [Ideas]
@mmgoodnow it would be great to send some coin your way for your work on this excellent application.

**Comment by mmgoodnow** [comment]
I appreciate the sentiment, but I make enough money. If you want to help out, the number 1 thing you can do is PR into https://github.com/cross-seed/cross-seed.org and make improvements to the documentation. (or make improvements to the real code, I welcome both!)
